app/src/main/java/org/wikipedia/feed/onthisday/OnThisDayFragment.kt

```diff
@@ -236,14 +237,14 @@ class OnThisDayFragment : Fragment(), CustomDatePicker.Callback {
         }
     }
 
-    private inner class EventsViewHolder(v: View, private val wiki: WikiSite) : RecyclerView.ViewHolder(v) {
+    private inner class EventsViewHolder(v: ViewOnThisDayEventBinding, private val wiki: WikiSite) : RecyclerView.ViewHolder(v.root) {
 
-        private val descTextView: TextView = v.findViewById(R.id.text)
-        private val yearTextView: TextView = v.findViewById(R.id.year)
-        private val yearsInfoTextView: TextView = v.findViewById(R.id.years_text)
-        private val pagesViewPager: ViewPager2 = v.findViewById(R.id.pages_pager)
-        private val pagesIndicator: TabLayout = v.findViewById(R.id.pages_indicator)
-        private val radioButtonImageView: ImageView = v.findViewById(R.id.radio_image_view)
+        private val descTextView: TextView = v.text
+        private val yearTextView: TextView = v.year
+        private val yearsInfoTextView: TextView = v.yearsText
+        private val pagesViewPager: ViewPager2 = v.pagesPager
+        private val pagesIndicator: TabLayout = v.pagesIndicator
+        private val radioButtonImageView: ImageView = v.radioImageView
```

Comment:
If we use the view binding on the view holder, that means we don't need to create `val` for the view items and we can use them directly. For example, `binding.text`.